Revamp Tyk APIs page to Tyk APIs Hub with improved layout (#4908)
TykTechnologies/tyk-docs: tyk-docs/content/tyk-apis.md

## Tyk Postman Collections

Explore our official Tyk Postman collections for a hands-on, practical approach to using our APIs.
Postman collections provide an interactive way to explore and test Tyk APIs. They are designed to help you quickly get
started and understand the capabilities of our products.

{{< grid >}}

{{< badge
    href="https://www.postman.com/tyk-technologies/workspace/379673ec-4cc5-4b8e-bef5-8a6a988071cb/overview"
    title="Postman" titleStyle="justify-content: center"
    image="img/logos/Postman-logo-vertical-orange-2021.svg" imageStyle="object-fit:contain" alt="Postman Logo"
>}}
<h4>Tyk Postman workspace</h4>
{{< /badge >}}

{{< /grid >}}

## Tyk Management - APIs Documentations

Access detailed OpenAPI documentation to understand the full capabilities of Tyk products. Our documentation provides
all the information you need to integrate Tyk APIs seamlessly into your projects.

{{< grid >}}

{{< badge href="tyk-dashboard-api" title="OpenAPI"
titleStyle="justify-content: center"
image="img/logos/tyk-logo-selfmanaged.svg" imageStyle="object-fit:scale-down" alt="Tyk Dashboard Logo">}}

> API to manage your API platform

Reviewer: Styling not displayed as centred for badge title, not sure why, I have checked and the styling seems to match the open source badge

Author: not sure i understand

---
title: Tyk APIs Hub

> description: Tyk API hub. In this section you will find OpenAPI docs for Tyk API Gateway, Tyk Dashboard and other Tyk products

Reviewer 2: ```suggestion
description: The Tyk API Hub. In this section you will find OpenAPI docs for the APIs for Tyk Gateway, Tyk Dashboard and other Tyk products
```

tags: ["swagger", "REST", "Tyk APIs", "Tyk OpenAPI Specs APIs", "OPenAPI", "OAS" ]
aliases:
  - /tyk-oas/
---

> In this page you will find the full range of Tyk APIs, designed to help you integrate and manage your API ecosystem

Reviewer 2: ```suggestion
All Tyk components are designed with an API that allows you to configure and control the product. On this page you will find documentation for all of these APIs, designed to help you to integrate Tyk into your software stack and then manage your API ecosystem efficiently.
```

> efficiently. Explore our comprehensive API documentation and Postman collections to get started quickly.

Reviewer 2: ```suggestion
```

This feels a little unnecessary.

## Tyk Postman Collections

Explore our official Tyk Postman collections for a hands-on, practical approach to using our APIs.

> Postman collections provide an interactive way to explore and test Tyk APIs. They are designed to help you quickly get

Reviewer 2: ```suggestion
These provide an interactive way to explore and test Tyk components' APIs. They are designed to help you quickly get
```

started and understand the capabilities of our products.

{{< grid >}}

{{< badge
    href="https://www.postman.com/tyk-technologies/workspace/379673ec-4cc5-4b8e-bef5-8a6a988071cb/overview"
    title="Postman" titleStyle="justify-content: center"
    image="img/logos/Postman-logo-vertical-orange-2021.svg" imageStyle="object-fit:contain" alt="Postman Logo"
>}}

> Tyk Postman workspace

Reviewer 2: ```suggestion
Tyk Public Workspace at Postman
```

{{< /badge >}}

{{< /grid >}}

> ## Tyk Enterprise - APIs Documentations

Reviewer 2: ```suggestion
## Tyk Enterprise
```

> Access detailed documentation to understand the full capabilities of Tyk products. Our documentation provides all the

Reviewer 2: ```suggestion
Here you can access detailed documentation enabling you to unlock the full capabilities of Tyk's enterprise products. This API documentation provides all the
```

> information you need to integrate Tyk Management APIs seamlessly into your projects.

Reviewer 2: ```suggestion
information you need to integrate Tyk's products seamlessly into your projects.
```

{{< grid >}}

{{< badge href="tyk-dashboard-api" title="OpenAPI"
titleStyle="justify-content: center"
image="img/logos/tyk-logo-selfmanaged.svg" imageStyle="object-fit:scale-down" alt="Tyk Dashboard Logo">}}

> Manage Tyk API platform

Reviewer 2: ```suggestion
Manage the Tyk platform
</br>from the Dashboard
```

{{< /badge >}}

{{< /grid >}}

{{< grid >}}

{{< badge href="dashboard-admin-api" title="OpenAPI" titleStyle="justify-content: center">}}
Dashboard Admin API
</br>

> </br>Provision and set up

Reviewer 2: ```suggestion
</br>Provision & administrate
```

</br>Tyk Dashboard
{{< /badge >}}

{{< badge href="tyk-mdcb-api" title="OpenAPI" titleStyle="justify-content: center">}}
MDCB API
</br>
</br>Monitor & diagnose

> </br>Tyk Data planes

Reviewer 2: ```suggestion
</br>Tyk Data Planes
```

{{< /badge >}}

{{< badge href="tyk-portal-api" title="OpenAPI" titleStyle="justify-content: center">}}
Enterprise Portal API
</br>

> </br>Managing and controlling

Reviewer 2: ```suggestion
</br>Configure & control
```

</br> Tyk Enterprise</br>Developer Portal
{{< /badge >}}

{{< /grid >}}

> ## Tyk Open Source - API Documentations

Reviewer 2: ```suggestion
## Tyk Open Source
```

{{< grid >}}

{{< badge href="tyk-gateway-api" title="OpenAPI"
titleStyle="justify-content: center"
image="img/logos/tyk-logo-opensource.svg" imageStyle="object-fit:scale-down" alt="Tyk Gateway Logo">}}

> Configure & Control

Reviewer 2: ```suggestion
Configure & control
</br>Open Source deployment
```
